game/plugins/src/main/kotlin/gg/rsmod/plugins/api/cfg/Npcs.kt

```diff
@@ -4260,12 +4260,12 @@ object Npcs {
     const val NISHA = 4648
     const val TYRAS_GUARD_4649 = 4649
     const val TRADER_STAN = 4650
-    const val TRADER_CREWMEMBER = 4651
-    const val TRADER_CREWMEMBER_4652 = 4652
-    const val TRADER_CREWMEMBER_4653 = 4653
-    const val TRADER_CREWMEMBER_4654 = 4654
-    const val TRADER_CREWMEMBER_4655 = 4655
-    const val TRADER_CREWMEMBER_4656 = 4656
+    const val TRADER_CREWMEMBER_BRIMHAVEN = 4651
+    const val TRADER_CREWMEMBER_SHIPYARD = 4652
+    const val TRADER_CREWMEMBER_CATHERBY = 4653
+    const val TRADER_CREWMEMBER_PORT_KHAZARD = 4654
+    const val TRADER_CREWMEMBER_MUSAPOINT = 4655
```

Comment:
If you can add the id number tags at the end of the names for all 6, it would be helpful. :)